frontend/src/pages/Policies.tsx

```diff
@@ -0,0 +1,270 @@
+import React, { ReactElement } from 'react';
+import { Container, Link, Typography, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({
+  h2: {
+    fontWeight: 600,
+    fontSize: '1.875em',
+    marginBottom: '0.4em',
+    '@media only screen and (max-width: 425px) ': {
+      fontSize: '1.5em',
+    },
+  },
+  h3: {
+    fontWeight: 500,
+    fontSize: '1.5em',
+    marginBottom: '0.4em',
+    '@media only screen and (max-width: 425px) ': {
+      fontSize: '1.25em',
+    },
+  },
+  h4: {
+    fontWeight: 500,
+    fontSize: '1.125em',
+    marginBottom: '0.4em',
+    '@media only screen and (max-width: 425px) ': {
+      fontSize: '0.938em',
+    },
+  },
+  body: {
+    fontFamily: 'Work Sans, sans-serif',
+    color: 'black',
+    textAlign: 'left',
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    fontSize: '0.938em',
+    '@media only screen and (max-width: 425px) ': {
+      fontSize: '0.75em',
+    },
+    lineHeight: '20px',
+    marginBottom: '1em',
+  },
+  link: {
+    color: '#B94630',
+  },
+}));
+
+const Policies = (): ReactElement => {
+  const { h2, h3, h4, body, link } = useStyles();
+  return (
+    <Container maxWidth="lg">
+      <Typography variant="h2" className={h2}>
```

Comment:
This is just a suggestion, but would this look better if we centered the title? I'm not sure if the privacy policy needs to follow a certain format though, so it's fine if we can't do this.

Reply:
I'll keep it as is to stay consistent with the rest of the website since that is all left-aligned!